tools/github-actions/cascading/src/helpers.ts

```diff
@@ -0,0 +1,81 @@
+import {BaseLogger} from './cascading';
+
+/**
+ * Ensure that the sorting based on git have been done correctly, update the list if there is an issue
```

Comment:
it's a very convoluted way to simply say "Sort the branches according to semantic versioning"